src/views/add-collaborator.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.html" %}
+
+{% block beforeContent %}
+  {% include "include/back-link.html" %}
+{% endblock %}
+
+{% block pageContent %}
+  <h1 class="govuk-heading-l">Add a New Collaborator to a Repository</h1>
```

Comment:
This should exactly match the title on the home page --> Add a new collaborator to a repository